tests/test_api.py

```diff
@@ -107,6 +110,61 @@ def test_api_transmute_tarball_to_conda_v2(testing_workdir):
     assert os.path.isfile(os.path.join(testing_workdir, test_package_name + '.conda'))
 
 
+@pytest.mark.skipif(sys.platform=="win32", reason="windows and symlinks are not great")
```

Comment:
General question for all of us (cc @jezdez @beeankha @rchord), how tightly should we be coupling our code? We have a number of util values/functions that are present throughout our repos and used both in the code and testing. E.g. for `on_win`/`sys.platform=="win32"` (and flavors thereof) should we be defaulting to using `on_win` at all times or should we be following some other metric for when/when not?